Dockerfile.microshift

```diff
@@ -0,0 +1,16 @@
+# This dockerfile is specific to building Multus for OpenShift
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS rhel9
+ADD . /usr/src/plugins
+WORKDIR /usr/src/plugins
+ENV CGO_ENABLED=0
```

Comment:
CGO is required to link against FIPS-compliant openssl lib.

Reply:
That is correct. Currently Dockerfile does not set but OCP builds overwride the configuration, hence it is kinds of Okey. But recommend that CGO should be enabled.

Reply:
Updated, thanks!